public/app/features/transformers/docs/content.ts

```diff
@@ -515,6 +516,7 @@ Use this transformation to customize the output of a string field. This transfor
 - **Kebab case** - Formats all characters in the string in lowercase and uses dashes instead of spaces between words.
 - **Trim** - Removes all leading and trailing spaces from the string.
 - **Substring** - Returns a substring of the string, using the specified start and end positions.
+- **Affix** - Add a prefix or suffix string to the string. Allows pasting the value of other fields when provided {field} as the affix expression.
```

Comment:
Wondering if there's a typo in the second sentence here. Should it be:
```suggestion
- **Affix** - Add a prefix or suffix string to the string. Allows pasting the value of other fields when provided {field} is the affix expression.
```

Reply:
Hi, the original is right, but should have read 

Add a prefix or suffix string to the string. Allows pasting the value of other fields, with the expression {field}.

I have pushed some other changes and reworked this text in the latest commit.